streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -320,48 +324,29 @@ public void shouldPauseActivePartitions() {
         replay();
 
         taskManager.setAssignmentMetadata(taskId0Assignment, Collections.<TaskId, Set<TopicPartition>>emptyMap());
+        taskManager.setPartitionsToTaskId(taskId0PartitionToTaskId);
         taskManager.createTasks(taskId0Partitions);
         verify(consumer);
     }
 
     @Test
     public void shouldSuspendActiveTasks() {
-        expect(active.suspend()).andReturn(null);
+        expect(active.revokeTasks(revokedTasks, revokedChangelogs)).andReturn(null);
+        expect(restoreConsumer.assignment()).andReturn(Collections.emptySet());
         replay();
 
-        taskManager.suspendTasksAndState();
+        taskManager.suspendActiveTasksAndState(revokedPartitions);
         verify(active);
     }
 
-    @Test
-    public void shouldSuspendStandbyTasks() {
-        expect(standby.suspend()).andReturn(null);
-        replay();
-
-        taskManager.suspendTasksAndState();
-        verify(standby);
-    }
-
-    @Test
-    public void shouldUnassignChangelogPartitionsOnSuspend() {
```

Comment:
This is related to the logic that we delay removal of the changelog topic -- not sure why we should do so though.

Reply:
Went back to removing changelogs on suspend, will add this test back in